data-reconciliation-app/docs/adr/01-data-ingest.md

```diff
@@ -0,0 +1,139 @@
+# Data Ingest API
+
+## Status
+
+Proposed
+
+## Context
+
+We need to build an API that ingests member data into our system. Once ingested, we also need to store this data in the K-V store.
+
+Gathered requirements:
+
+- Members will upload data via csv
+- Members will upload multiple records (batch) or single records via csv
+- Data will have one attribute, and we will only work with string attributes for now.
+
+## Decision
+
+### Endpoint
+
+```
+POST  /csv
```

Comment:
Is this endpoint actually posting the data or voting? If that is voting, could this be called `POST /votes`?

Reply:
added to latest commit